.nvmrc

```diff
@@ -0,0 +1 @@
+v16.8
```

Comment:
Nit: please add a new line at the end of the file

Reply:
Ah good point!  Done.